lib/pavilion/commands/result.py

```diff
@@ -100,6 +100,13 @@ def _setup_arguments(self, parser):
                  "default, 'all' will only display tests newer than 1 day ago, but setting any "
                  "filter argument will override that."
         )
+        parser.add_argument(
+            '-d', '--num-digits', dest="num_digits",
+            help="The maximum number of digits to display when formatting floating point values. If"
+                 " set, numbers much larger or much smaller than 1 are displayed in scientific "
+                 "notation with the specified precision, and numbers near 1 are truncated. If not "
+                 "provided, defaults to standard Python rules for formatting numeric values."""
+        )
```

Comment:
I think this should actually default to 5.

I also think it should be called 'max-digits'.